src/modules/listCard.js

```diff
@@ -1,31 +1,31 @@
-import { getLocalStorage } from "./data.js";
+import { getLocalStorage } from './data.js';
 
-const toDoList = document.querySelector(".toDoContainer");
+export const toDoList = document.querySelector('.toDoContainer');
 
 export const displayListTasks = (tasksArray) => {
-  toDoList.innerHTML = "";
+  toDoList.innerHTML = '';
 
   tasksArray = getLocalStorage();
   tasksArray.forEach((toDo) => {
```

Comment:
- [ ] One test is failing, kindly fix this failing test as this is a requirement.
![image](https://user-images.githubusercontent.com/81932478/214816570-da8b5c7e-5731-4113-9878-b30e25209ca0.png)